scripts/convertSituation/convertTo2Tonnes.ts

```diff
@@ -0,0 +1,85 @@
+import NGCRules from '@incubateur-ademe/nosgestesclimat/public/co2-model.FR-lang.fr.json'
+import type { NodeValue } from '@incubateur-ademe/nosgestesclimat'
+import path from 'path'
+import Engine from 'publicodes'
+import { readYAML } from '../utils'
+import type { Situation } from '../../src/types/types'
+
+type Props = {
+  situation: Situation
+}
+
+export default function convertTo2Tonnes({ situation }: Props) {
+  const conversionRulesFile = path.join(
+    process.cwd(),
+    'scripts/convertSituation/2Tonnes/conversion.publicodes'
+  )
+  const conversionRules = readYAML(conversionRulesFile)
+
+  const default2TFile = path.join(
+    process.cwd(),
+    'scripts/convertSituation/2Tonnes/default2T.yaml'
+  )
+
+  const default2T = readYAML(default2TFile)
+
+  const ifAbsent2TFile = path.join(
+    process.cwd(),
+    'scripts/convertSituation/2Tonnes/ifAbsent2T.yaml'
+  )
+  const ifAbsent2T = readYAML(ifAbsent2TFile)
+
+  // We need to merge NGCRules and conversionRules to have all the rules in the same engine
+  const rules = { ...NGCRules, ...conversionRules }
+  const engine = new Engine(rules)
```

Comment:
You already have an engine with the french model in consts

Reply:
We can't add rules to an engine already instantiated